src/mixins/filing-template-mixin.ts

```diff
@@ -758,6 +758,10 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
         folioNumber: this.getFolioNumber, // default FN; may be overwritten by staff BCOL FN
         isFutureEffective: false
       },
+      business: {
+        legalType: this.getEntityType,
+        identifier: this.getTempId
+      },
```

Comment:
Filings UI [now checks for this object](https://github.com/bcgov/business-filings-ui/blob/9ac2527f4a87117777134f0e4479181e89122617/src/App.vue#L607) since it's [potentially de-referenced](https://github.com/bcgov/business-filings-ui/blob/9ac2527f4a87117777134f0e4479181e89122617/src/App.vue#L730).

So, Create UI had better save this object!

(The other filings already save a business object.)